app/controllers/orders_controller.rb

```diff
@@ -1,6 +1,7 @@
 class OrdersController < ApplicationController
   def show
     @order = Order.find(params[:id])
+    @items = @order.order_items
```

Comment:
user items in the view, I couldn't get the @order.items.each to work, I think this could be refactored to an includes maybe? Ideas?